Updated GitHub Campus Program copy for Doc page (#6735)
github/docs: content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/about-github-campus-program.md

{% data variables.product.prodname_campus_program %} is a package of premium {% data variables.product.prodname_dotcom %} access for teaching-focused institutions that grant degrees, diplomas, or certificates. {% data variables.product.prodname_campus_program %} includes:

- No-cost access to {% data variables.product.prodname_ghe_cloud %} and {% data variables.product.prodname_ghe_server %} for all of your technical and academic departments

> - 50,000 GitHub Actions minutes and 50GB GitHub Packages storage

Reviewer: Added product variables and a space between `50` and `GB` for an easier read.

```suggestion
- 50,000 {% data variables.product.prodname_actions %} minutes and 50 GB {% data variables.product.prodname_registry %} storage
```

> - 50,000 GitHub Actions minutes and 50GB GitHub Packages storage

Reviewer: Added product variables and a space between `50` and `GB` for an easier read.

```suggestion
- 50,000 {% data variables.product.prodname_actions %} minutes and 50 GB {% data variables.product.prodname_registry %} storage
```

- Exclusive access to new features, GitHub Education-specific swag, and free developer tools from {% data variables.product.prodname_dotcom %} partners
- Automated access to premium {% data variables.product.prodname_education %} features, like the {% data variables.product.prodname_student_pack %}

To read about how GitHub is used by educators, see [GitHub Education stories.](https://education.github.com/stories)

### {% data variables.product.prodname_campus_program %} terms and conditions

- The license is free for one year and will automatically renew for free every 2 years. You may continue on the free license so long as you continue to operate within the terms of the agreement. Any school that can agree to the [terms of the program](https://education.github.com/schools/terms) is welcome to join.

- Please note that the licenses are for use by the whole school. Internal IT departments, academic research groups, collaborators, students, and other non-academic departments are eligible to use the licenses so long as they are not making a profit from its use. Externally funded research groups that are housed at the university may not use the free licenses.

> - You must offer GitHub to all of your technical and academic departments and your school’s logo will be shared on the GitHub Education website as a GitHub Campus Program Partner.

Reviewer: Adding some product variables

```suggestion
- You must offer {% data variables.product.prodname_dotcom %} to all of your technical and academic departments and your school’s logo will be shared on the GitHub Education website as a {% data variables.product.prodname_campus_program %} Partner.
```

> - New organizations in your enterprise are automatically added to your enterprise account. To add organizations that existed before your school joined the GitHub Campus Program, please contact [GitHub Education Support](https://support.github.com/contact/education). For more information about administrating your enterprise, see "[Setting up and managing your enterprise](/github/setting-up-and-managing-your-enterprise)." New organizations in your enterprise are automatically added to your enterprise account. To add organizations that existed before your school joined the GitHub Campus Program, please contact GitHub Education Support. 

Reviewer: ```suggestion
- New organizations in your enterprise are automatically added to your enterprise account. To add organizations that existed before your school joined the {% data variables.product.prodname_campus_program %}, please contact [GitHub Education Support](https://support.github.com/contact/education). For more information about administrating your enterprise, see "[Setting up and managing your enterprise](/github/setting-up-and-managing-your-enterprise)." New organizations in your enterprise are automatically added to your enterprise account. To add organizations that existed before your school joined the {% data variables.product.prodname_campus_program %}, please contact GitHub Education Support. 
```

> To read more about GitHub's privacy practices, see ["Global Privacy Practices"](https://docs.github.com/en/github/site-policy/global-privacy-practices)

Reviewer: Link fix + product variable
```suggestion
To read more about {% data variables.product.prodname_dotcom %}'s privacy practices, see ["Global Privacy Practices"](/github/site-policy/global-privacy-practices)
```

### {% data variables.product.prodname_campus_program %} Application Eligibility

- Often times, a campus CTO/CIO, Dean, Department Chair, or Technology Officer signs the terms of the program on behalf of the campus.

> - If your school does not issue email addresses, GitHub will reach out to your account administrators with an alternative option to allow you to distribute the student developer pack to your students.

Reviewer: Product variable
```suggestion
- If your school does not issue email addresses, {% data variables.product.prodname_dotcom %} will reach out to your account administrators with an alternative option to allow you to distribute the student developer pack to your students.
```

For more information, see the [official {% data variables.product.prodname_campus_program %}](https://education.github.com/schools) page.

> If you're a student or academic faculty and your school isn't partnered with {% data variables.product.prodname_dotcom %} as a {% data variables.product.prodname_campus_program %} school, then you can still individually apply for discounts to use {% data variables.product.prodname_dotcom %}. To apply for the Student Developer Pack, [see the application form.](https://education.github.com/pack/join)

Reviewer: ```suggestion
If you're a student or academic faculty and your school isn't partnered with {% data variables.product.prodname_dotcom %} as a {% data variables.product.prodname_campus_program %} school, then you can still individually apply for discounts to use {% data variables.product.prodname_dotcom %}. To apply for the Student Developer Pack, [see the application form](https://education.github.com/pack/join).
```
